client/src/elements/molecules/item-list-entry.js

```diff
@@ -26,12 +30,34 @@ export class ItemListEntry {
     });
   }
 
+  toggleOptionDropdown() {
+    this.optionsDropdown.classList.toggle(
+      "item-list-entry__options-dropdown--show"
+    );
+  }
+
+  activateItem() {
+    this.item.activate();
+  }
+
+  deactivateItem() {
+    this.item.deactivate();
+  }
+
   deleteItem() {
     this.item.delete().then(() => {
-      this.element.addEventListener("transitionend", () => {
-        this.element.parentNode.removeChild(this.element);
-      });
-      this.element.style.transform = "scale(0)";
+      this.element.parentNode.removeChild(this.element);
     });
   }
+
+  async blueprint() {
+    try {
+      await this.item.blueprint();
+      this.item.conf.title = this.i18n.tr("item.blueprintTitlePrefix");
+      let tool = this.item.conf.tool.replace(new RegExp("-", "g"), "_");
```

Comment:
the replacement of `-` with `_` can go away. it's a relict from old times where tools were named with `-` causing some problems with the search.
It can also go away from item-overview.js where this is copied from.

would be nice to think about a way to somehow abstract these functions so we do not have duplicate code here and in item-overview.

maybe some kind of shared ItemActionController object that can be injected.

Reply:
The `item-overview` page and the `item-list-entry` component use now the `ItemActionController` for activate, deactivate, edit, delete and blueprint actions -> https://github.com/nzzdev/Q-editor/pull/133/commits/41a453fed75808388217eea5fe4ff9341ea01561